app/assets/javascripts/discourse/app/components/sidebar/switch-panel-buttons.js

```diff
@@ -0,0 +1,24 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+
+export default class SwitchPanelButtons extends Component {
+  @service router;
+
+  constructor() {
+    super(...arguments);
+    this.setCurrentPanelKey = this.args.setCurrentPanelKey;
```

Comment:
I'm not sure if this is needed considering that we can access the args in any method and also by `@setCurrentPanelKey` in the template.